openmp/runtime/src/kmp_lock.cpp

```diff
@@ -3431,6 +3431,9 @@ void __kmp_cleanup_indirect_user_locks() {
                     ll));
       __kmp_free(ll->lock);
       ll->lock = NULL;
+      // reset the reverse critical section pointer to 0
+      if (ll->rev_ptr_critSec && LIKELY(!__kmp_in_atexit))
```

Comment:
```suggestion
      if (ll->rev_ptr_critSec && !UNLIKELY(__kmp_in_atexit))
```
Avoids adding new `LIKELY`